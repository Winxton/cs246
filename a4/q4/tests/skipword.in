w_manywords.txt skipword
w_manywords.txt skipword skipword
