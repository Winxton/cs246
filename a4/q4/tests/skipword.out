b
d
f
123
789
z
(*!!
d
123
z
